docs/sources/user/Output-format-l2tcsv.md

```diff
@@ -0,0 +1,36 @@
+# l2tcsv output format
+
+By default log2timeline.pl used the l2tcsv (or l2t_csv) output format. l2tcsv
```

Comment:
I would make it clearer that this is not the case anymore.

Reply:
ack, added note.